lib/appConfig.ts

```diff
@@ -18,7 +19,13 @@ export const appName = process.env.npm_package_name
 export const appVersion = process.env.npm_package_version
 export const appNameSanitized = appName.replace(/[/\\]/, '-')
 
-export interface AppOptions extends BaseOptions {
+export interface AppOptions extends Omit<BaseOptions, 'inlineCSS'> {
+	/**
+	 * Inject all styles inside the javascript bundle instead of emitting a .css file
+	 * @default false
+	 */
+	inlineCSS?: boolean | VitePluginInjectCSSOptions,
```

Comment:
Is there a case there we would like to inline CSS in an App, not a lib?

Reply:
That is currently the default for webpack built apps, so probably yes?

Reply:
Indeed 🥲

Reply:
Why the sad face? This is quite ok, no?
Saves us to manually add the stylesheet with `addStyle` as well as a few bytes of additional requests (and reduce the loading queue)

Reply:
> Why the sad face? This is quite ok, no?

It is worse in a page performance. One more request doesn't change loading as much (if it is not HTTP/1, it is likely a parallel request) as loading more JS (not in parallel), parsing JS code and executing it. Which doesn't replace parsing and applying css, it is an additional expensive blocking task. So an app launches later.

CSS stored in JS also has more size. 50kb css ~= 200kb css in a js bundle on a small experiment with Talk. 

And adding styles in runtime increases the number of layout operations in the rendering pipeline, making the initial rendering slower (there are no app styles from the initial load), but it is unlikely noticeable on our load with a super large JS.

`style-loader` is usually used for development because it is faster on bundling and HMR (and on small libs).

Reply:
How would you do that for libs then?
That means we'd have to import the css manually all the time too?

Reply:
You could disable injecting the CSS, vite can handle multiple entry points and will produce one CSS file for each entry point.
So if your app provides lets say `main.js` and `settings.js`, you will also get `main.css` and `settings.css`.